spec/lib/os_places_api/client_spec.rb

```diff
@@ -182,6 +182,62 @@
       end
     end
 
+    context "the postcode is outdated in the database" do
+      before do
+        Postcode.create(
+          postcode: postcode,
+          results: [
+            {
+              "DPA" => {
+                "UPRN" => "6714279",
+                "UDPRN" => "54673874",
+                "ADDRESS" => "1, WHITECHAPEL HIGH STREET, LONDON, E1 8QS",
+                "BUILDING_NUMBER" => "1",
+                "THOROUGHFARE_NAME" => "WHITECHAPEL HIGH STREET",
+                "POST_TOWN" => "LONDON",
+                "POSTCODE" => "E1 8QS",
+                "RPC" => "2",
+                "X_COORDINATE" => 533_813.0,
+                "Y_COORDINATE" => 181_262.0,
+                "LNG" => -0.072, # should be updated
+                "LAT" => 51.51, # should be updated
+                "STATUS" => "APPROVED",
+                "LOGICAL_STATUS_CODE" => "1",
+                "CLASSIFICATION_CODE" => "CO01",
+                "CLASSIFICATION_CODE_DESCRIPTION" => "Office / Work Studio",
+                "LOCAL_CUSTODIAN_CODE" => 5900,
+                "LOCAL_CUSTODIAN_CODE_DESCRIPTION" => "TOWER HAMLETS",
+                "POSTAL_ADDRESS_CODE" => "D",
+                "POSTAL_ADDRESS_CODE_DESCRIPTION" => "A record which is linked to PAF",
+                "BLPU_STATE_CODE" => "1",
+                "BLPU_STATE_CODE_DESCRIPTION" => "Under construction",
+                "TOPOGRAPHY_LAYER_TOID" => "osgb1000006035651",
+                "LAST_UPDATE_DATE" => "17/06/2017",
+                "ENTRY_DATE" => "17/02/2017",
+                "BLPU_STATE_DATE" => "17/02/2017",
+                "LANGUAGE" => "EN",
+                "MATCH" => 1.0,
+                "MATCH_DESCRIPTION" => "EXACT",
+              },
+            },
+          ],
+        )
+      end
+
+      it "should query OS Places API and update cached data" do
```

Comment:
```suggestion
      it "should query OS Places API and update cached data when `update: true` is passed" do
```

I'd also add another test with `update: false` passed, that checks that the data _isn't_ updated (or rather that no HTTP requests are made to OS Places API).